package.json

```diff
@@ -82,6 +82,7 @@
     "@nextcloud/eslint-config": "^8.3.0",
     "@nextcloud/stylelint-config": "^2.4.0",
     "@nextcloud/webpack-vue-config": "^6.0.1",
+    "@types/crypto-js": "^4.2.2",
```

Comment:
Why are you adding it? 
According to their repo, there is a native Crypto in browsers now 
https://github.com/brix/crypto-js#discontinued

Reply:
Then it's time to migrate to native methods)